### PR TITLE
Gaming Convention: The main and the about pages

### PR DESCRIPTION
**Pull request by ha-manel**
## Hi there! 👋🏻 
🎯 in this project, I:
- Added a folder of the necessary images for the project.
- Created the HTML documents for the `main` page and the `about` page of the website.
- Styled the pages using Sass.
- Created sections dynamically using JavaScript (the `program cards` and the `featured speakers`).
- Created the redundant components using JavaScript.

Here is a [loom video](https://www.loom.com/share/a52b5e7115034bdd8f5db9939db95cfd) of me presenting the project.
Cheers 😄

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <p class="sub-text">Take a look at the last two Gaming Conventions which took place in Tokyo and NewYork</p>
+    <div id="past-events-imgs">
+      <a href="#" class="event" id="event-1"><span>2019</span><span>GC 2019 in Tokyo</span></a>
+      <a href="#" class="event" id="event-2"><span>2017</span><span>GC 2019 in NewYork</span></a>
+    </div>
+  </section>
+
+  <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+  <script src="./scripts/page-header.js"></script>
+  <script src="./scripts/sponsors-footer.js"></script>
```
Comment (gemmen29) [inline about.html:52]: - [ ] Kindly change the footer color in the desktop version as it's light in the Behance design on the home page.

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
-# Microverse-Gaming-Convention
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (gemmen29) [inline README.md:1]: - [ ] Kindly try to fix the overlapping problem in the mobile version when I press the more button one of the speakers overlaps the brand's section.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @ha-manel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
Hi @ha-manel 👋, 

## STATUS: APPROVED ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**